tasks/bower-concat.js

```diff
@@ -35,10 +35,11 @@ module.exports = function(grunt) {
 		var callback = this.data.callback;
 		var bowerOptions = this.data.bowerOptions || {};
 		var bowerDir = bowerOptions.relative !== false ? bower.config.cwd : '';
+		var options = this.options();
 
 		var done = this.async();
 		bowerMainFiles(function(jsFiles, cssFiles) {
-			concatenateAndWriteFile(jsFiles, jsDest);
+			concatenateAndWriteFile(jsFiles, jsDest, true);
```

Comment:
`concatenateAndWriteFile(jsFiles, jsDest, options.separator);`

No need for an extra verbose option.